### DISCUSSION TITLE
Change in name of cQube?

### DISCUSSION THREAD

**Discussion by karsood** [Sunbird-cQube/community#60] [General]
Hello team,

I wanted to check if the name of cQube is being changed to something else? If so, what is the new name and can that be reflected in the sunbird.org website?

**Comment by SreenivasNimmagadda** [comment]
@karsood,

As of now, there is no official announcement of changing the cQube name. 

But we are in the development of another application that will be a part of cQube umbrella. But, still, another application is in the development stage only.

**Comment by RNarmatha** [comment]
Replied
